### DISCUSSION TITLE
自己保存的conversation是否可以删除

### DISCUSSION THREAD

**Discussion by kring61** [timqian/openprompt.co#40] [General]
自己保存的conversation是否可以删除，只是测试了一下相应的prompt

**Comment by timqian** [comment]
删除，star都会有的，不过还在开发，可以先发一下我 conversation 的链接我来删除一下

**Reply by iriedaisuki** [reply]
请您把我尝试保存的也删除了吧！非常感谢！https://openprompt.co/ChatGPT/conversations/61

**Reply by timqian** [reply]
已删除～

**Reply by iriedaisuki** [reply]
非常感谢！

**Comment by kring61** [comment]
https://openprompt.co/to-chinese/conversations/39

麻烦您啦~上面是我的conversation的地址



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Tim ***@***.***&gt;; 
发送时间: 2023年4月5日(星期三) 上午10:47
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [timqian/openprompt.co] 自己保存的conversation是否可以删除 (Discussion #40)





 
删除，star都会有的，不过还在开发，可以先发一下我 conversation 的链接我来删除一下

**Reply by timqian** [reply]
已删除

**Comment by chenxuetao** [comment]
https://openprompt.co/ChatGPT/conversations/154 求删，感谢~

**Reply by timqian** [reply]
已删～

**Reply by chenxuetao** [reply]
感谢感谢！

**Comment by Shuxuyangguang** [comment]
https://openprompt.co/conversations/397 求删 太谢谢啦~

**Reply by timqian** [reply]
已删

**Comment by timqian** [comment]
<img width="936" alt="Screen Shot 2023-04-09 at 15 35 27" src="https://user-images.githubusercontent.com/5512552/230760385-60aab0e6-2e45-4867-9775-fe4ec589ef9e.png">
已支持删除和编辑 conversation title
